.changeset/clever-eagles-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-adr': patch
```

Comment:
Woops - just noticed that `adr-common` should also be in the list. I will fix that in a followup